Tools/_sunrise/pr_check_summary.py

```
@@ -0,0 +1,743 @@
+
+def is_test_row(row: CheckRow) -> bool:
+    return bool(row.failure and row.failure.test_names) or "test" in row.name.lower()
+
+
+def status_icon(status: str) -> str:
+    if status == "success":
+        return "✅"
+    if status == "failure":
+        return "❌"
+    if status in {"missing", "skipped"}:
+        return "➖"
```

Ruff (0.15.11)

[warning] String contains ambiguous `➖` (HEAVY MINUS SIGN). Did you mean `-` (HYPHEN-MINUS)?

(RUF001)